### ISSUE TITLE
Magento 2 container creation

### ISSUE THREAD

**Issue by Sanchi10** [magento/magento2-samples#49]
m not getting proper steps to create a container for my own theme in magento2

**Comment by peterjaap** [comment]
I don't think this is the place to ask this type of question. I'd suggest you head over to the [Magento Stackexchange](http://magento.stackexchange.com) and try it there.
